force-app/main/default/classes/Main.cls

```diff
@@ -0,0 +1,28 @@
+Person p = new Person();
```

Comment:
I didn't go over this, but you should add this file to the .gitignore file.

https://www.atlassian.com/git/tutorials/saving-changes/gitignore

Reply:
We'd want to do this b/c it's not pertinent to the solution(s).